web/docs/advanced/accessing-app-config.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Accessing the configuration 
+---
+
+Whenever you start a Waps app, you are starting two processes.
+ - **The client process** - A React app that implements your app's frontend.
```

Comment:
```suggestion
 - **The client process** - A React app that implements your app's client.
```

Reply:
This was intentional.

The bullet starts with "The client process" and then explains it by using the term "frontend", making sure that:
- Everyone finds the terminology they're familiar with.
- We don't self-reference terms in their explanation.
- We connect the frontend terminology with the client before we mention our env var `frontendUrl` (this is why we mention the offical name first and emphasize it with bold text).